Samplesheet update (#227)
sanger-tol/blobtoolkit: docs/output.md

- `busco/`

>   - `<lineage-name>/`

Reviewer: ```suggestion
  - `<lineage>/`
```

>     - `<assembly>.<lineage>.short_summary.json`: BUSCO scores for that lineage as JSON.

Author: Currently in our snapshots is "_", I thought it is to distinguish between the dot in fasta id and normal '.'

Reviewer: I'll update the snapshots.

No, it's just an oversight from my side in the other PRs.
In the GAP convention, `.` is used to separate fields

- `read_mapping/`
  - `<datatype>/`
    - `<specimen>/<run>/`
      - `<assembly>.<datatype>.<specimen>.<run>.minimap2.bam`: alignments of that sample's reads in BAM format.

>       - `<assembly>.<datatype>.<specimen>.<run>.minimap2.bam.csi`: index of the BAM file.

Reviewer: ```suggestion
    - `<sample>/`  (can be nested directories if `sample` contains `/`)
      - `<assembly>.<datatype>.<sample>.minimap2.bam`: alignments of that sample's reads in BAM format. (`/` in the sample name are replaced with `.`)
      - `<assembly>.<datatype>.<sample>.minimap2.bam.csi`: index of the BAM file. (`/` in the sample name are replaced with `.`)
```

>       - `<assembly>.<datatype>.<specimen>.<run>.<aligner>.bam.csi`: index of the BAM file.

Reviewer: ```suggestion
      - `<assembly>.<datatype>.<specimen>.<run>.minimap2.bam`: alignments of that sample's reads in BAM format.
      - `<assembly>.<datatype>.<specimen>.<run>.minimap2.bam.csi`: index of the BAM file.
```

</details>

### Read coverage

Read coverage statistics as computed by the pipeline.
Those files are the raw data used to build the BlobDir.
The `<window_size>` is formatted as `<N>k` when the window size is divisible by 1000 (for example `1k`) and `<N>bp` otherwise (for example `1500bp`).

<details markdown="1">
<summary>Output files</summary>

- `read_mapping/`
  - `<datatype>/`
    - `<specimen>/<run>/` (or `<sample>/` if this information is provided in the samplesheet)
      - `<assembly>.<datatype>.<sample>.coverage.<window_size>.bed.gz`: BedGraph file with per-window coverage.

>       - `<assembly>.<datatype>.<specimen>.<run>.minimap2.coverage.<window_size>.bed.gz`: same as above when alignments are generated by this pipeline using minimap2 (`--align`).

Reviewer: ```suggestion
    - `<sample>/`  (can be nested directories if `sample` contains `/`)
      - `<assembly>.<datatype>.<sample>.coverage.<window_size>.bed.gz`: BedGraph file with per-window coverage. (`/` in the sample name are replaced with `.`)
      - `<assembly>.<datatype>.<specimen>.<run>.minimap2.coverage.<window_size>.bed.gz`: same as above when alignments are generated by this pipeline using minimap2 (`--align`). (`/` in the sample name are replaced with `.`)
```

>       - `<assembly>.<datatype>.(<specimen>.<run>|<sample>).minimap2.coverage.<window_size>.bed.gz`: same as above when alignments are generated by this pipeline using minimap2 (`--align`).

Reviewer: ```suggestion
      - `<assembly>.<datatype>.<specimen>.<run>.minimap2.coverage.<window_size>.bed.gz`: same as above when alignments are generated by this pipeline using minimap2 (`--align`).
```

</details>

### Base content

_k_-mer statistics.
Those files are the raw data used to build the BlobDir.
The `<window-size>` follows the same formatting as in Read coverage.

<details markdown="1">
<summary>Output files</summary>

- `base_content/`
  - `k1/`, `k2/`, `k3/`, `k4/`
    - `<assembly>.<mono|di|tri|tetranuc>.<window-size>.tsv.gz`: tab-separated files with counts of every _k_-mer for k &le; 4. The first three columns correspond to the coordinates (sequence name, start, end), followed by each _k_-mer.
  - `<assembly>.freq.<window_size>.tsv.gz`: tab-separated file with frequencies derived from the _k_-mer counts.

> 

Reviewer: The docs are correct
